src/dashboard/src/main/models.py

```diff
@@ -691,3 +837,45 @@ class Meta:
 
     def __unicode__(self):
         return self.term
+
+class WatchedDirectory(models.Model):
+    id = UUIDPkField()
+    watched_directory_path = models.TextField(null=True, blank=True, db_column='watchedDirectoryPath')
+    chain = models.ForeignKey('MicroServiceChain', null=True, db_column='chain')
+    only_act_on_directories = models.IntegerField(default=1, db_column='onlyActOnDirectories')
```

Comment:
I'll swap it to `BooleanField`; we've never used `NULL` values in this despite it being allowed in the schema.